infinispan-remote/src/main/java/org/hibernate/ogm/datastore/infinispanremote/InfinispanRemoteProperties.java

```diff
@@ -12,6 +12,13 @@
 /**
  * Properties for configuring the Infinispan Remote datastore via {@code persistence.xml} or
  * {@link StandardServiceRegistryBuilder}.
+ * <p>
+ * It is possible to configure the Hot Rod (Infinispan remote) client prefixing them with 'hibernate.ogm.infinispan_remote.client.'.
+ * For example, the property 'infinispan.client.hotrod.tcp_no_delay' can be set in the hibernate.properties file
+ * using 'hibernate.ogm.infinispan_remote.client.infinispan.client.hotrod.tcp_no_delay'.
```

Comment:
I agree with you and there are two reasons why I opted for the prefix:
1. I don't have to list all the properties for hot rod
2. the user doesn't have to learn different too many different rules to set the same property.

I will try a different approach to see what happens